pkgs/games/warzone2100/default.nix

```diff
@@ -11,17 +12,25 @@ let
   };
 in
 
-stdenv.mkDerivation rec {
-  version = "3.2.3";
-  name = "${pname}-${version}";
+mkDerivation rec {
+  inherit pname;
+  version  = "3.3.0";
+  checkout = "20190831-182824-924bf1e";
```

Comment:
Is this actually used?

Reply:
yeah, it's a leftover from the previous source url.